### ISSUE TITLE
Concurrent transfers

### ISSUE THREAD

**Issue by qwadratic** [MinaFoundation/mina-fungible-token#41]
Currently, submitting two transfer transactions from the same account will lead to approval of only first transaction, because nonce precondition is explicitly set on the `from` token account

**Comment by kantp** [comment]
I don't think this is the case anymore with the current implementation; looking at a transfer transaction, I only see a nonce for the `feePayer`.
